symfony/plugins/orangehrmMaintenancePlugin/config/orangehrmMaintenancePluginConfiguration.class.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * OrangeHRM Enterprise is a closed sourced comprehensive Human Resource Management (HRM)
+ * System that captures all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM Inc is the owner of the patent, copyright, trade secrets, trademarks and any
+ * other intellectual property rights which subsist in the Licensed Materials. OrangeHRM Inc
+ * is the owner of the media / downloaded OrangeHRM Enterprise software files on which the
+ * Licensed Materials are received. Title to the Licensed Materials and media shall remain
+ * vested in OrangeHRM Inc. For the avoidance of doubt title and all intellectual property
+ * rights to any design, new software, new protocol, new interface, enhancement, update,
+ * derivative works, revised screen text or any other items that OrangeHRM Inc creates for
+ * Customer shall remain vested in OrangeHRM Inc. Any rights not expressly granted herein are
+ * reserved to OrangeHRM Inc.
+ *
+ * Please refer http://www.orangehrm.com/Files/OrangeHRM_Commercial_License.pdf for the license which includes terms and conditions on using this software.
+ *
+ */
+class orangehrmMaintenancePluginConfiguration extends sfPluginConfiguration {
+
+    public function initialize() {
+        $enabledModules = sfConfig::get('sf_enabled_modules');
+        if (is_array($enabledModules)) {
+            sfConfig::set('sf_enabled_modules', array_merge($enabledModules, array('maintenance')));
+        }
+    }
+}
```

Comment:
Make sure you configure the ID to add new line when save the file

Reply:
done